correct url in import

Travis ci failed with

```
> make test
go get golang.org/x/crypto/sha3
go get github.com/jbenet/go-base58
go test -race -cpu=5 -v ./...
multihash/main.go:9:2: cannot find package "github.com/jbenet/go-multihash" in any of:
    /home/travis/.gimme/versions/go1.3.linux.amd64/src/pkg/github.com/jbenet/go-multihash (from $GOROOT)
    /home/travis/gopath/src/github.com/jbenet/go-multihash (from $GOPATH)
multihash/main.go:10:2: cannot find package "github.com/jbenet/go-multihash/opts" in any of:
    /home/travis/.gimme/versions/go1.3.linux.amd64/src/pkg/github.com/jbenet/go-multihash/opts (from $GOROOT)
    /home/travis/gopath/src/github.com/jbenet/go-multihash/opts (from $GOPATH)
make: *** [go_test] Error 1
```

github.com/jbenet/go-multihash should be github.com/multiformats/go-multihash

--- a/multihash/README.md
+++ b/multihash/README.md
@@ -2,7 +2,7 @@
 
 The `multihash` tool uses `go-multihash` to hash things much like `shasum`.
 
-Warning: this is a **multihash** tool! Its digests follow the [multihash](https://github.com/jbenet/multihash) format.
+Warning: this is a **multihash** tool! Its digests follow the [multihash](https://github.com/multiformats/multihash) format.
 
 ### Install
 

